lib/fs/basicfs_test.go

```diff
@@ -33,15 +33,15 @@ func TestChmodFile(t *testing.T) {
 	path := filepath.Join(dir, "file")
 	defer os.RemoveAll(dir)
 
-	defer os.Chmod(path, 0666)
+	defer os.Chmod(path, 0666) //skipcq
```

Comment:
Rather than adding these I think we should just tell deepsource to ignore lax permissions in test files

Reply:
Or move the permissions value to a variable

Reply:
> For stub methods it's obvious that the parameters are unused, because it's a method stub without a body. Having the parameter names makes the code more natural and readable.

Agreed, but is `//skipcq` acceptable to quiet these? In test files only, or all files?

> For Chtimes in fakefs for example I _don't_ like the change because "_ time.Time, mtime time.Time` just tells me there's some other time passed that we apparently don't care about, but leaves it a mystery what that time is and why we ignore it.

Understood. Restore `atime` but add `//skipcq` to quiet?

> Rather than adding these I think we should just tell deepsource to ignore lax permissions in test files

Sorry, but how? DeepSource's docs show quieting specific lines of code, or ignoring entire files. I don't see a way to ignore specific errors for entire files. I've asked DeepSource if this is possible on their Slack #ideas channel.

> Or move the permissions value to a variable.

Good idea. We'll see if that works.

Note that our [contributing page](https://docs.syncthing.net/dev/contributing.html#tests) may need to be tweaked a bit, as it currently says `That the output from go lint and go vet is clean.` Based on your feedback, I can submit a PR to freshen this statement to clarify what is being asked of our developers.